tests/lib/nested.sh

```diff
@@ -1196,35 +1196,33 @@ nested_start_core_vm_unit() {
         else
             PARAM_BIOS="-bios /usr/share/ovmf/OVMF.fd"
         fi
-        local OVMF_CODE OVMF_VARS
-        OVMF_CODE="secboot"
-        OVMF_VARS="ms"
+        local OVMF_CODE OVMF_VARS OVMF_SIZE
+        OVMF_CODE=""
+        OVMF_VARS=""
+        OVMF_SIZE=""
```

Comment:
Where does this get set to another value?

Reply:
I forgot to remove it, initially it was set to 4M in noble, but it doesn't work

Reply:
updated